lib/Server.js

```diff
@@ -126,6 +131,20 @@ class Server {
     return path.resolve(dir, "node_modules/.cache/webpack-dev-server");
   }
 
+  apply(compiler) {
+    this.compiler = compiler;
+
+    let started = false;
+
+    compiler.hooks.watchRun.tapPromise(pluginName, async () => {
```

Comment:
We should enable `watch` before taping into `watchRun` hook.

Reply:
Yes, plugin should work only for `webpack watch`/`webpack serve`, but I am afraid it is so late to override fs, need think more about it

Reply:
ping @alexander-akait

Reply:
Yep, in my roadmap on the next week